lib/movie_presentation_image__and_title.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
+import 'styles.dart';
+
+class MoviePresentationImageTitle extends StatelessWidget {
+  const MoviePresentationImageTitle({
+    super.key,
+    required this.borderRadiusBackDrop,
+    required this.movieBackDrop,
+    required this.imageHeightBackDrop,
+    required this.sizeofOriginalTitleBox,
+    required this.leftPaddingImageOriginalTitle,
+    required this.movieOriginalTitle,
+    required this.originalTitleMaxLines,
+  });
+
+  final double borderRadiusBackDrop;
+  final String movieBackDrop;
+  final double imageHeightBackDrop;
+  final double sizeofOriginalTitleBox;
+  final double leftPaddingImageOriginalTitle;
+  final String movieOriginalTitle;
+  final int originalTitleMaxLines;
```

Comment:
Are all of this parameters really needed? I mean, does this widget have to be so customizable? Just asking, let's discuss about it.

Reply:
You could even provide some default values for them, to not force the ending user to specify all of them at once, just what they need to change (like most Flutter's widgets!)

Reply:
I fixed it! They are constants from the parent!